GRIB2_Template_4_34_ProductDefinitionTemplate_en.csv

```diff
@@ -1,35 +1,35 @@
+Title_en,OctetNo,Contents_en,Note_en,noteIDs,codeTable,flagTable,Status
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",10,Parameter category,(see Code table 4.1),,4.1,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",11,Parameter number,(see Code table 4.2),,4.2,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",12,Type of generating process,(see Code table 4.3),,4.3,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",13,Background generating process identifier (defined by originating centre),,,,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",14,Analysis or forecast generating process identifier,,,,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",15-16,Hours of observational data cut-off after reference time,(see Note 1),33,,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",17,Minutes of observational data cut-off after reference time,,,,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",18,Indicator of unit of time range,(see Code table 4.4),,4.4,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",19-22,Forecast time in units defined by octet 18,(see Note 2),111,,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",23,Number of contributing spectral bands (NB),,,,,Operational
+"individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer, in a continuous or non-continuous interval for simulated (synthetic) satellite data",,"24-   Repeat the following 11 octets for each contributing band (nb = 1, NB)",,,,,Operational
```

Comment:
@amilan17 more than one space here "24- Repeat"

Reply:
leaving as is. other tables follow the same pattern